src/control/SConscript

```diff
@@ -33,6 +33,20 @@ def get_install_bin_path(benv, name):
     """Get the final installation location for a binary with a given name"""
     return join(benv.subst("$PREFIX"), "bin", name)
 
+def is_firmware_mgmt_build(benv):
+    "Check whether this build has firmware management enabled."
+    fw_mgmt_enabled = benv.subst("$FIRMWARE_MGMT")
+    return (fw_mgmt_enabled and
```

Comment:
You might consider making this a BoolVariable or an EnumVariable to give clarity in the help output for how to set it or unset it.   Could have ['on', 'off'] with default 'off' or boolean with default False

https://scons.org/doc/3.1.2/HTML/scons-api/SCons.Variables.BoolVariable'-module.html

Reply:
Thanks, Jeff. Always appreciate your scons expertise. I've updated it in the latest commit.